src/Components/SalesforceWrapper.tsx

```diff
@@ -58,7 +58,7 @@ export const SalesforceWrapper: React.FC = () => {
     },
   })
 
-  window.embedded_svc?.showHelpButton?.()
+  window?.embedded_svc?.showHelpButton?.()
```

Comment:
Saw this moved to the effect where its guaranteed to be defined, but note that the way to check this is
```
if (typeof window !== "undefined")
```
existential operator will still error out